src/picongpu/CMakeLists.txt

```diff
@@ -268,6 +268,18 @@ if( (Boost_VERSION EQUAL 105500) AND
       "${CUDA_NVCC_FLAGS} \"-DBOOST_NOINLINE=__attribute__((noinline))\" ")
 endif()
 
+################################################################################
+# OPENMP
+##############################################################################
+
+FIND_PACKAGE(OpenMP)
+IF(NOT OPENMP_FOUND)
+    MESSAGE(STATUS "Disable OpenMP: Can't find library")
+
+ELSE()
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
```

Comment:
note for later `target`-redesign: [can be added](http://cmake.3232098.n2.nabble.com/link-errors-using-OpenMP-tp7581939p7581970.html) to `COMMPILE_FLAGS` and `LINK_FLAGS`